src/Router.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import Navigation from './Navigation';
+import Home from './routes/Home';
+import Chat from './routes/Chat';
+import Profile from './routes/Profile';
+import LogIn from './routes/LogIn';
+import ChatList from './routes/ChatList';
+import styled from 'styled-components';
+
+const Router = ({ isLoggedIn, user, friends }) => {
+  return (
+    <Appwrapper>
+      <BrowserRouter>
+        {isLoggedIn && <Navigation />}
+        <Switch>
+          {isLoggedIn ? (
+            <InteractingSpaceWrapper>
+              <Route exact path={`/`}>
+                <Home friends={friends} />
+              </Route>
```

Comment:
```suggestion
              <Route 
                exact 
                path={`/`} 
                render={ ({friends}) => { <Home friends={friends} />} } 
              />
```
이런식으로 `Route` 컴포넌트 내부에 `render` prop을 사용하면 컴포넌트를 self-closing 하게 만들수 있습니다